packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -1848,6 +1733,16 @@ class ReviewSectionComp extends React.Component<FullProps, State> {
       ).length
     )
 
+    const reviewSection = formSections.find(({ viewType }) =>
+      ['preview', 'review'].includes(viewType)
+    )
+
+    if (!reviewSection) {
+      throw new Error(
+        'The form definition should always contain a review or preview section'
+      )
+    }
```

Comment:
also food for thought, should the user ever flow to ReviewSection if the form definition doesn't contain something required? Kind of an architectural problem as here the validation flows quite far from the actual form definitions or the place where we fetch the forms.